packages/gestalt/src/SegmentedControl.js

```diff
@@ -9,12 +9,16 @@ import styles from './SegmentedControl.css';
 type Props = {|
   items: Array<React.Node>,
   onChange: ({ event: SyntheticMouseEvent<>, activeIndex: number }) => void,
+  responsive?: boolean,
   selectedItemIndex: number,
   size?: 'md' | 'lg',
 |};
 
 export default function SegmentedControl(props: Props) {
-  const { items, onChange, selectedItemIndex, size = 'md' } = props;
+  const { items, onChange, responsive, selectedItemIndex, size = 'md' } = props;
+  const buttonStyle = responsive
+    ? undefined
+    : { width: `${Math.floor(100 / Math.max(1, items.length))}%` };
```

Comment:
i would make `style` always present on the div and instead move the ternary to `width: responsive ? undefined : '...'`

Reply:
Good catch. FWIW, the reason I did this is because I wanted to keep the default the same and all the snapshots unchanged, but this is no longer relevant. Will update.